seqr/views/apis/staff_api_2_3_tests.py

```diff
@@ -788,6 +803,33 @@ def test_upload_qc_pipeline_output(self, mock_file_iter):
         self.assertDictEqual(indiv.pop_platform_filters, {'n_insertion': '38051', 'r_insertion_deletion': '1.8064E+00'})
         self.assertEqual(indiv.population, 'OTH')
 
+    @mock.patch('seqr.views.apis.staff_api.file_iter')
+    def test_upload_sv_qc(self, mock_file_iter):
+        url = reverse(upload_qc_pipeline_output, )
+        self.check_staff_login(url)
+
+        request_data = json.dumps({
+            'file': 'gs://seqr-datasets/v02/GRCh38/RDG_WES_Broad_Internal/v15/sample_qc/sv/sv_sample_metadata.tsv'
+        })
+
+        mock_file_iter.return_value = SAMPLE_SV_QC_DATA
+        response = self.client.post(url, content_type='application/json', data=request_data)
+        self.assertEqual(response.status_code, 200)
+        response_json = response.json()
+        self.assertSetEqual(set(response_json.keys()), {'info', 'errors', 'warnings'})
+        self.assertListEqual(response_json['info'], [
+            'Parsed 6 SV samples',
+            'Found and updated matching seqr individuals for 4 samples'
+        ])
+        self.assertListEqual(response_json['warnings'], ['The following 2 samples were skipped: MANZ_1169_DNA, NA'])
+
+        self.assertIsNone(Individual.objects.get(individual_id='NA19675_1').sv_flags)
+        self.assertListEqual(Individual.objects.get(individual_id='NA19678').sv_flags, ['high_QS_rare_calls:_>10'])
+        self.assertListEqual(Individual.objects.get(individual_id='HG00732').sv_flags, ['raw_calls:_>100'])
+        self.assertListEqual(
+            Individual.objects.get(individual_id='HG00733').sv_flags,
+            ['raw_calls:_>100', 'high_QS_rare_calls:_>10'])
```

Comment:
Is it better to use a `set` comparison?

Reply:
no, they are put it in as an ordered list and so there should be no way for the order to ever change